mod/ping.php

```diff
@@ -344,6 +344,12 @@ function ping_init(&$a) {
 	killme();
 }
 
+/**
+ * @brief Retrieves the notifications array for the given user ID
+ *
+ * @param int $uid
```

Comment:
` @param int $uid The user ID`
I know I'm a little bit nit-picking. But as far as it is is possible we should always add a description after the variables/arrays/objects. Please check this also for the other doxygen blocks (also for `@return`). Some times it is really self explanatory but it would rise the consistency and would lower the the barrier for real newcomers